android/app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +29,7 @@
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
       <meta-data
         android:name="com.google.android.geo.API_KEY"
-        android:value="AIzaSyDPM51h9M14rGYM43QPRVQ8szi_qkMM168"/>
+        android:value="AIzaSyCUGR9KL3PBR4cqf45QFWmkTdwDp5fALBE"/>
```

Comment:
Yoojeong used her own api key the first time linked to her personal email. Now, it is using an api key linked to the geo challenges email she set up for the project.